email_notification/templates/email/html/partials/base.jinja

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="{{ language }}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{% block title %}Varaamo{% endblock %}</title>
+
+    <style>
+        @font-face {
+            font-family: "HelsinkiGrotesk";
+            src: url("https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/565d73a693abe0776c801607ac28f0bf.woff") format("woff");
+        }
+
+        body {
+            background-color: #FFFFFF;
+            max-width: 960px;
```

Comment:
Apparently not supported in Window Outlook: https://www.caniemail.com/features/css-max-width/